templates/schema.json

```diff
@@ -11,6 +11,10 @@
       "type": "string",
       "description": "Longer description of the template."
     },
+    "category": {
+      "type": "string",
+      "description": "Optional category to use when showing a list of templates."
```

Comment:
I've been trying to set [FUTURE RELEASE] on these until there is anything implemented in Saturn to support.